documentation/docs/get-started/setup.mdx

```diff
@@ -134,7 +134,7 @@ python -m pip install khoj-assistant
  # CPU
  python -m pip install khoj-assistant
  # NVIDIA (CUDA) GPU
- CMAKE_ARGS="DLLAMA_CUBLAS=on" FORCE_CMAKE=1 python -m pip install khoj-assistant
```

Comment:
What the previous argument incorrect? Or what effect would it have had?

Reply:
This is just an update based on the [fixed documentation of llama-cpp-python](https://github.com/abetlen/llama-cpp-python?tab=readme-ov-file#supported-backends)